### DISCUSSION TITLE
[BUG] Server Crash

### DISCUSSION THREAD

**Discussion by ThePureCC** [GameServerManagers/LinuxGSM#3496] [General]
## Basic info

* **Distro:** [CentOS 7]
* **Game:** [Rust]
* **Command:** [n/a]
* **LinuxGSM version:** [latest]

## Further Information

Game Server randomly crashed. Everything was fine, and it just suddenly crashed. I don't know what exactly happened. The only thing was that there was a weird Steam Invalid login from a player right before it crashed. I've attached the part of the log in pastebin:

https://pastebin.com/xKAWkRX3

I don't know if this is the right place to ask for help, but this is the only place I know. It does look like the game server crashed and not linuxgsm. I'd really appreciate it if someone could shed some light on what happened or how to prevent something like this from happening.

**Comment by Claiyc** [comment]
Definitely not the right place to ask as this is an issue with rust and not with lgsm. Expect this issue to be closed without a proper answer. However you can try your luck on the lgsm discord, there are plenty of people willing to help out with gameserver specific issues.

**Comment by h3o66** [comment]
i dont think we can solve that crash anyway, so i guess so i think this would be better one the discord or discussions instead of a lgsm bug
